i18n/fr/docusaurus-plugin-content-docs/version-22.04/migrate/migrate-from-el-to-debian.md

```diff
@@ -0,0 +1,235 @@
+   chown centreon-broker: /var/lib/centreon/status/*
+   chown centreon-gorgone: /var/lib/centreon/nagios-perf/perfmon-* -R
+   chown centreon-engine: /var/lib/centreon/centplugins/*
+   ```
+
+### Étape 3 : Récupérer les bases de données
+
+1. Sur l'ancien serveur, faitez un dump des bases de données :
+
+   ```shell
+   mysqldump -u root -p centreon > /tmp/centreon.sql
+   mysqldump -u root -p centreon_storage > /tmp/centreon_storage.sql
+   ```
+
+2. Sur l'ancien serveur, arrêtez MariaDB :
+
+   ```shell
+   service mysqld stop
+   ```
+
+3. Depuis l'ancien serveur, exportez les dumps vers le nouveau serveur de base de données (assurez-vous d'avoir assez d'espace):
+
+   ```shell
+   rsync -avz /tmp/centreon.sql root@<IP_NOUVEAU_CENTREON>:/tmp/
+   rsync -avz /tmp/centreon_storage.sql root@<IP_NOUVEAU_CENTREON>:/tmp/
+   ```
+
+4. Sur le nouveau serveur de base de données, supprimez les bases de
+   données vierges et recréez-les :
+
+   ```shell
+   mysql -u root -p
+   ```
+
+   ```SQL
+   DROP DATABASE centreon;
+   DROP DATABASE centreon_storage;
+   CREATE DATABASE centreon;
+   CREATE DATABASE centreon_storage;
+   ```
+
+5. Sur le nouveau serveur de bases de données, importez dans la base les dumps précédemment transférés :
+
+   ```shell
+   mysql -u root centreon -p </tmp/centreon.sql
+   mysql -u root centreon_storage -p </tmp/centreon_storage.sql
+   ```
+
+6. Exécutez l'upgrade des tables :
+
+   ```shell
+   mysql_upgrade
+   ```
+
+   Si votre base de données est protégée par mot de passe, entrez :
+
+   ```shell
+   mysql_upgrade -u <utilisateur_admin_bdd> -p
+   ```
+
+   Exemple : si votre utilisateur_admin_bdd est `root`, entrez:
+
+   ```shell
+   mysql_upgrade -u root -p
+   ```
+
+7. Démarrez le processus **mariadb** sur le nouveau serveur :
+
+   ```shell
+   systemctl start mariadb
+   ```
+
+> Remplacez **\<IP_NOUVEAU_CENTREON\>** par l'adresse IP de votre nouveau serveur
+> Centreon.
+
+### Étape 4 : Synchroniser les plugins
+
+Si vous n'utilisez que des plugins Centreon, réinstallez-les sur le nouveau serveur:
+
+```shell
+apt update
+echo "deb https://apt.centreon.com/repository/22.04-plugin-packs/ bullseye main" >> /etc/apt/sources.list.d/centreon-pp.list
+wget -O- https://apt-key.centreon.com | gpg --dearmor | tee /etc/apt/trusted.gpg.d/centreon.gpg > /dev/null 2>&1
+apt update
+apt install centreon-pack*
+apt install centreon-plugin-\*
+```
+
+Si vous utilisez vos propres plugins personnalisés, synchronisez les répertoires qui contiennent ceux-ci, ainsi que toutes éventuelles dépendances.
+
+### Étape 5 : Montée de version de la solution Centreon
+
+1. Sur le nouveau serveur, forcez la montée de version en déplacant le contenu du répertoire
+   **/var/lib/centreon/installs/install-22.04.0-YYYYMMDD_HHMMSS** dans le
+   répertoire **/usr/share/centreon/www/install** :
+
+   ```shell
+   cd /var/lib/centreon/installs/
+   mv install-22.04.0-YYYYMMDD_HHMMSS/ /usr/share/centreon/www/install/
+   ```
```

Comment:
what version to use if older machine had an older 22.04 than the new one ?
idea: using the same version as the old one, to be sure to fully match the older machine conf and DB requests